spec/file_translation/fts_mt.yaml

```diff
@@ -8,10 +8,6 @@ x-paths:
       tags:
         - File Machine Translations (MT)
       operationId: mtFile
-      consumes:
```

Comment:
`consumes` and `produces` no longer exist in `openapi v3`

Reply:
`consumes` was replaced by `requestBody` and `produces` described in `responses`